conda/run_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+export LANG=C
+
+cd "${PREFIX}/share/gadgetron/test/integration" || exit 1
+
+python get_data.py
+python run_tests.py cases/*
```

Comment:
I think we may have to be deliberate about which cases we run. I think this command will just skip tests where requirements are not met, but we may want it to fail if GPUs are present, but we get a requirements not met from Gadgetron

Reply:
I believe the test runner skips the matlab and julia tests within this environment (should it be... missing runtime deps?). 

Where do you see such a check living? My initial thought is to expand the Gadgetron capability query within the test infrastructure itself to perform this validation check. Though we would need to define what 'GPU capable' means. For instance you could have a gpu but the CUDA runtime is too old. Putting it here would mean that the check would run both within the dev and package environment (and any other build environment developers of the gadgetron leverage). 

On a side note this feels like a distinct test that differs from the typical test cases. So I could also see run_tests.py invoking a second script (similar to how run_gadgetron_test.py is invoked) for non case related tests (such as capability query). We could consider implementing these tests using pytest or other frameworks.

Reply:
The problem here is the `run_gadgetron_test.py` will check if the gadgetron is able to run the test and otherwise skip. That can lead us to falsely assuming that all the tests pass when actually it is just skipping them because it would just skip a GPU test if we didn't compile correctly and don't have those libraries. 

For that reason the test scripts have an `--ignore-requirements` where you can say `--ignore-requirements cuda` to make sure the CUDA tests always run (https://github.com/gadgetron/gadgetron/blob/master/test/integration/run_tests.py#L277). For the docker images we then pass that in when we test: https://github.com/gadgetron/gadgetron/blob/master/azure-pipelines.yml#L51. We should have something similar here so we can test this package on a platform with GPU and make sure it works. We should insist those tests run so that they fail if CUDA support doesn't work.